File: Selenium/Activities/Activity2.java
package activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {

	public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();

        try {
              driver.get("https://training-support.net/webelements/login-form/");
              System.out.println("Page Title: " + driver.getTitle());
              driver.findElement(By.id("username")).sendKeys("admin");
              driver.findElement(By.id("password")).sendKeys("password");
              driver.findElement(By.xpath("//button[text()='Log in']")).click();

        } finally {
            driver.quit();
        }
    }
}

File: Selenium/Activities/activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


driver = webdriver.Firefox()

try:
    driver.get("https://training-support.net/webelements/login-form/")

    print("Page Title:", driver.title)

    wait = WebDriverWait(driver, 10)

    username = wait.until(EC.presence_of_element_located((By.ID, "username")))
    username.send_keys("admin")

    password = driver.find_element(By.ID, "password")
    password.send_keys("password")

    login_btn = wait.until(EC.element_to_be_clickable((By.TAG_NAME, "button")))
    login_btn.click()

finally:
    driver.quit()